scripts/report_event.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+###############################################################################
+# vim: tabstop=4:shiftwidth=4:expandtab:
+# Copyright (c) 2017 SIOS Technology Corp. All rights reserved.
+#
+# Arguments:
+#   args[1] - Windows Event Source/Provider 
+#   args[2] - Windows Event ID
+#   args[3] - Windows Event Severity
+#   args[4] - Windows Event Message
+#   args[5] - Windows Event Time Generated in ISO 8601 format (2017-10-11T15:18:33-0500)
+##############################################################################
+"""
+This script will send three iQ events (one for each layer type) for a single Windows event.
+"""
+
+import logging
+import sys
+from datetime import datetime, timedelta
+from time import tzname, localtime, strftime
+from pytz import timezone
+from os.path import dirname, realpath
+
+curr_path = dirname(realpath(__file__))
+sys.path.insert(0, '{}/../../'.format(curr_path))
+
+from SignaliQ.client import Client
+from SignaliQ.model.CloudProviderEvent import CloudProviderEvent
+from SignaliQ.model.ProviderEventsUpdateMessage import ProviderEventsUpdateMessage
+
+__log__ = logging.getLogger(__name__)
```

Comment:
If not used, this can be removed

Reply:
Which, line 33? I went by the example in the SDK. You do this in both example.py and minigun.py. I thought it was left in for ease of use later? If it is unnecessary I can remove it.